client/src/components/TaskCard/TaskCard.js

```diff
@@ -21,6 +21,10 @@ const TaskCard = ({
     setModal(!showModal);
   };
 
+  const toggleReflection = () => {
```

Comment:
To make it a bit more descriptive, I would rename it to `toggleReflectionModal`

Reply:
Renamed.